packages/vue/README.md

```diff
@@ -0,0 +1,126 @@
+| `speed`                 | number                |          | 1                     | Animation playback speed. 1 is regular speed.                                                                                                                                                                                                      |
+| `data`                  | string \| ArrayBuffer |          | undefined             | Animation data provided either as a Lottie JSON string or as an ArrayBuffer for .lottie animations.                                                                                                                                                |
+| `mode`                  | string                |          | "forward"             | Animation play mode. Accepts "forward", "reverse", "bounce", "bounce-reverse".                                                                                                                                                                     |
+| `backgroundColor`       | string                |          | undefined             | Background color of the canvas. Accepts 6-digit or 8-digit hex color string (e.g., "#000000", "#000000FF"),                                                                                                                                        |
+| `segments`              | \[number, number]     |          | \[0, totalFrames - 1] | Animation segments. Accepts an array of two numbers, where the first number is the start frame and the second number is the end frame.                                                                                                             |
+| `renderConfig`          | RenderConfig          |          | `{}`                  | Configuration for rendering the animation.                                                                                                                                                                                                         |
+| `useFrameInterpolation` | boolean               |          | false                 | Determines if the animation should update on subframes. If set to false, the original AE frame rate will be maintained. If set to true, it will refresh at each requestAnimationFrame, including intermediate values. The default setting is true. |
+
+### Listening to [DotLottie](../web/README.md) Events
+
+```vue
+<script setup>
+import { onMounted, ref, watch } from 'vue';
+import { DotLottieVue } from '@lottiefiles/dotlottie-vue'
+const player = ref(null);
+
+onMounted(() => {
+  if (player.value) {
+    player.value.addEventListener('pause', () => {
+      console.log('paused')
+    });
+    player.value.addEventListener('frame', ({ currentFrame }) => {
+      console.log('Frame:', currentFrame)
+    });
+  }
+})
+</script>
+
+<template>
+  <DotLottieVue autoplay loop ref="player" src="path-to-lottie.lottie" />
+</template>
+```
```

Comment:
we need to update this example
```suggestion
```vue
<script setup>
import { onMounted, ref, watch } from 'vue';
import { DotLottieVue } from '@lottiefiles/dotlottie-vue'
const playerRef = ref(null);
const dotLottie = playerRef?.value.getDotLottieInstance();

onMounted(() => {
    dotLottie.addEventListener('pause', () => {
      console.log('paused')
    });
    dotLottie.addEventListener('frame', ({ currentFrame }) => {
      console.log('Frame:', currentFrame)
    });
  }
})
</script>

<template>
  <DotLottieVue autoplay loop ref="player" src="path-to-lottie.lottie" />
</template>
```
```